package/docker/openmrs/Dockerfile

```diff
@@ -82,12 +82,15 @@ COPY package/resources/blank-user.png /etc/bahmni/
 
 # Used by envsubst command for replacing environment values at runtime
 RUN yum install -y \
-    mysql \
-    mysql-client \
     gettext-base \
     gettext \
     acl
 
+# Install mysql client to run sql scripts
```

Comment:
@mohan-13 Can we have the installation steps in 89 and 84 combined? Also a clean up of yum caches can reduce the image size and remove potential security risks from leftover package management files.

```
# Install dependencies and clean up
RUN yum install -y gettext-base gettext acl \
    && yum install -y https://dev.mysql.com/get/mysql80-community-release-el7-3.noarch.rpm \
    && rpm --import https://repo.mysql.com/RPM-GPG-KEY-mysql-2023 \
    && yum install -y mysql-community-client \
    && yum clean all \
    && rm -rf /var/cache/yum
```

Reply:
I agree with cleaning cache. But I would keep the installation steps separate as both are different dependencies and can be upgraded irrespective of other. Keeping it separate also allow docker image layers which optimises push and pull operations.